docs/en/integrations/dbt/dbt-incremental-model.md

```diff
@@ -0,0 +1,210 @@
+
+    Update the file `actor_summary.sql` with the following:
+
+    ```sql
+    {{ config(order_by='(updated_at, id, name)', engine='MergeTree()', materialized='incremental', 
+    unique_key='id') }}
+    ```
+
+    Note that our model will only respond to updates and additions to the `roles` and `actors` tables. To respond to all tables, users would be encouraged to split this model into multiple sub models - each with their own incremental criteria. These models can in turn be referenced and connected. For further details on cross referencing models see [here](https://docs.getdbt.com/reference/dbt-jinja-functions/ref).
+
+2. Execute a `dbt run` and confirm the results of the resulting table.
+
+    ```bash
+    clickhouse-user@clickhouse:~/imdb$  dbt run 
+    15:33:34  Running with dbt=1.0.4
+    15:33:34  Found 1 model, 0 tests, 1 snapshot, 0 analyses, 181 macros, 0 operations, 0 seed files, 6 sources, 0 exposures, 0 metrics
+    15:33:34  
+    15:33:35  Concurrency: 1 threads (target='dev')
+    15:33:35  
+    15:33:35  1 of 1 START incremental model imdb_dbt.actor_summary........................... [RUN]
+    15:33:41  1 of 1 OK created incremental model imdb_dbt.actor_summary...................... [OK in 6.33s]
+    15:33:41  
+    15:33:41  Finished running 1 incremental model in 7.30s.
+    15:33:41  
+    15:33:41  Completed successfully
+    15:33:41  
+    15:33:41  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
+    ```
+
+    ```sql
+    SELECT * FROM imdb_dbt.actor_summary ORDER BY num_movies DESC LIMIT 5;
+
+    +------+------------+----------+------------------+------+---------+-------------------+
+    |id    |name        |num_movies|avg_rank          |genres|directors|updated_at         |
+    +------+------------+----------+------------------+------+---------+-------------------+
+    |45332 |Mel Blanc   |832       |6.175853582979779 |18    |84       |2022-04-26 15:26:55|
+    |621468|Bess Flowers|659       |5.57727638854796  |19    |293      |2022-04-26 15:26:57|
+    |372839|Lee Phelps  |527       |5.032976449684617 |18    |261      |2022-04-26 15:26:56|
+    |283127|Tom London  |525       |2.8721716524875673|17    |203      |2022-04-26 15:26:56|
+    |356804|Bud Osborne |515       |2.0389507108727773|15    |149      |2022-04-26 15:26:56|
+    +------+------------+----------+------------------+------+---------+-------------------+
+    ```
+
+3. We will now add data to our model to illustrate an incremental update. Add our actor  "Clicky McClickHouse" to the `actors` table.
+
+    ```sql
+        INSERT INTO imdb.actors VALUES (845466, 'Clicky', 'McClickHouse', 'M');
```

Comment:
I'm a huge fan of Clicky McClickHouse's movies. Glad to finally see some recognition for his work!

Seriously, we need to reuse this name everywhere in the docs :) cc @tom-clickhouse

Reply:
will find someone with some artistic skill to bring this character to life